lib/recurly/client.php

```diff
@@ -62,6 +62,7 @@ class Recurly_Client
   const PATH_COUPON_REDEMPTION = '/redemption';
   const PATH_COUPON_REDEMPTIONS = '/redemptions';
   const PATH_COUPONS = '/coupons';
+  const PATH_GIFT_CARDS = '/gift_cards';
```

Comment:
i'm really not too sure... i guess having pieces that were easy to update all in one place? but the one time we tried to do that (coupon -> coupons) it ended up having lots of side effects. yeah i think if we want to encapsulate them having methods on classes for building URLs makes a lot more sense.